bagit.py

```diff
@@ -505,8 +505,10 @@ def _validate_entries(self, processes):
         if not available_hashers:
             raise RuntimeError("%s: Unable to validate bag contents: none of the hash algorithms in %s are supported!" % (self, self.algs))
 
-        def _init_worker():
-            signal.signal(signal.SIGINT, signal.SIG_IGN)
+        if os.name == 'posix':
+            worker_init = posix_multiprocessing_worker_initializer
```

Comment:
since `worker_init` is defined in these if else blocks will it exist on line 520?